http/rest-client-reactive/src/test/java/io/quarkus/ts/http/restclient/reactive/FileIT.java

```diff
@@ -0,0 +1,149 @@
+package io.quarkus.ts.http.restclient.reactive;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledOnOs;
+import org.junit.jupiter.api.condition.OS;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.QuarkusApplication;
+import io.restassured.response.Response;
+
+@QuarkusScenario
+public class FileIT {
+
+    private static final String BIGGER_THAN_TWO_GIGABYTES = OsUtils.SIZE_2049MiB;
+    private final Path downloaded;
+    private final Path uploaded;
+    private final OsUtils utils = OsUtils.get();
+
+    @QuarkusApplication(properties = "modern.properties")
+    static RestService app = new RestService().withProperties("modern.properties");
+
+    public FileIT() throws IOException {
```

Comment:
In this case, it it doesn't matter, so let's leave it as it is.
But when there are `@After...` methods, or inheritance between test classes, it may be better to stick to the JUnit convention. For better readability, mainly.